backend/rooms.py

```diff
@@ -162,41 +162,22 @@ def room_info(request, room_id):
     room_collection = "dm_rooms"
     current_room = DB.read(room_collection, {"_id": room_id})
 
-    if current_room and current_room.get("status_code", None) == None:
+    if current_room and current_room.get("status_code", None) is None:
 
         if "room_user_ids" in current_room:
             room_user_ids = current_room["room_user_ids"]
         elif "room_member_ids" in current_room:
             room_user_ids = current_room["room_member_ids"]
         else:
             room_user_ids = ""
-        if "starred" in current_room:
-            starred = current_room["starred"]
-        else:
-            starred = ""
-        if "pinned" in current_room:
-            pinned = current_room["pinned"]
-        else:
-            pinned = ""
-        if "bookmark" in current_room:
-            bookmark = current_room["bookmark"]
-        else:
-            bookmark = ""
-        if "private" in current_room:
-            private = current_room["private"]
-        else:
-            private = ""
-        if "created_at" in current_room:
-            created_at = current_room["created_at"]
-        else:
-            created_at = ""
+        starred = current_room["starred"] if "starred" in current_room else ""
+        pinned = current_room["pinned"] if "pinned" in current_room else ""
+        bookmark = current_room["bookmark"] if "bookmark" in current_room else ""
+        private = current_room["private"] if "private" in current_room else ""
+        created_at = current_room["created_at"] if "created_at" in current_room else ""
         if "org_id" in current_room:
             org_id = current_room["org_id"]
-        if "room_name" in current_room:
-            room_name = current_room["room_name"]
-        else:
-            room_name = ""
-
+        room_name = current_room["room_name"] if "room_name" in current_room else ""
```

Comment:
Here, can write something like this: 
`room_name = current_room['room_name'] or "".
If the first operand fails room_name takes ""`.
What do you think?

Reply:
Ohhh
I'd effect that

Reply:
Fixed to 
room_name = current_room.get("room_name", "")

Reply:
Oh yeah Great.